### ISSUE TITLE
Language override doesn't work

### ISSUE THREAD

**Issue by mrmokilok** [microsoft/BotFramework-WebChat#2847]
Setting language property on renderWebChat doesn't override browser selection anymore. This creates a problem, especially as the cognitive speech uses the same language.

**Comment by tdurnford** [comment]
Are passing the language to Web Chat through the `local` prop? Take a look at the [Change Local](https://github.com/microsoft/BotFramework-WebChat/tree/master/samples/02.branding-styling-and-customization/g.change-locale) Sample for more details.

```javascript
window.WebChat.renderWebChat(
  {
      directLine: window.WebChat.createDirectLine({
        token: 'YOUR_DIRECT_LINE_TOKEN'
      }),
      locale: 'en-US'
  },
  document.getElementById('webchat')
);
```

**Comment by corinagum** [comment]
You may also need to specify locale to Cognitive Services. Please take a look at our speech documentation: https://github.com/microsoft/BotFramework-WebChat/blob/master/docs/SPEECH.md

**Comment by mrmokilok** [comment]
Changing prop to "locale" from "language" worked. Btw, if the prop in vanilla js initialisation has been changed the instructions should be changed:
![image](https://user-images.githubusercontent.com/60223790/73047354-240f3800-3e7e-11ea-8316-c79f148e2994.png) (and probably the old prop should have a deprecation period)

@corinagum , the speech documentation says that the display language for  should override the browser locale (https://github.com/microsoft/BotFramework-WebChat/blob/master/docs/SPEECH.md#selecting-voice). I like this, because if I don't set specific voice, I get the benefit of constant development of new voices. 

I failed to mention, I'm using the latest branch, because renderMarkdown isn't supported in master.

Thank you for the very fast reply!
